### PR TITLE
Fix documentation for various modules

### PR DESCRIPTION
**Pull request by piskvorky**
Reviewing the remaining high-profile modules (utils, corpora, interfaces).

### CODE REVIEW

File: gensim/corpora/dictionary.py [hunk 1]
```diff
@@ -463,7 +458,7 @@ def save_as_text(self, fname, sort_by_word=True):
         fname : str
             Path to output file.
         sort_by_word : bool, optional
-            Sort words in lexicographical order before writing them out?
+            Sort words in lexicographical order before writing them out.
```
Comment (piskvorky) [inline gensim/corpora/dictionary.py]: -1: bool should be a question, to make it clear what positive answer means (True).

File: gensim/corpora/wikicorpus.py [hunk 2]
```diff
@@ -526,17 +510,15 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
...
         lemmatize : bool
-            Use lemmatization instead of simple regexp tokenization?
-
-            Defaults to `True` if you have the `pattern package <https://github.com/clips/pattern>`_ installed.
+            Use lemmatization instead of simple regexp tokenization.
+            Defaults to `True` if you have the `pattern <https://github.com/clips/pattern>`_ package installed.
         dictionary : :class:`~gensim.corpora.dictionary.Dictionary`, optional
             Dictionary, if not provided,  this scans the corpus once, to determine its vocabulary
-            (this needs **really long time**).
+            (**IMPORTANT: this needs really long time**).
```
Comment (piskvorky) [inline gensim/corpora/wikicorpus.py]: If it's important, it should be in brackets :)

`really long time` => `a really long time`

File: gensim/corpora/wikicorpus.py [hunk 3]
```diff
@@ -546,7 +528,11 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
         token_max_len : int, optional
             Maximal token length.
         lower : bool, optional
-             Convert all text to lower case?
+             Convert all text to lower case.
```
Comment (piskvorky) [inline gensim/corpora/wikicorpus.py]: -1: should be a question (here and elsewhere).

File: gensim/utils.py [hunk 4]
```diff
@@ -1260,12 +1259,12 @@ def chunkize(corpus, chunksize, maxsize=0, as_numpy=False):
...
             If > 0, prepare chunks in a background process, filling a chunk queue of size at most `maxsize`.
         as_numpy : bool, optional
-            Yield chunks as `np.ndarray`s instead of lists?
+            Yield chunks as `np.ndarray` instead of lists?
 
         Yields
         ------
         list OR np.ndarray
-            `chunksize`-ed chunks of elements from `corpus`.
+            "chunksize"-ed chunks of elements from `corpus`.
```
Comment (piskvorky) [inline gensim/utils.py]: Why this change? `chunksize` is a parameter name (just like ``corpus``, not `"corpus"`).

Comment (menshikh-iv) [reply gensim/utils.py]: in Sphinx construction
```
`something`other
```
without space between "`" and "o" break render

File: gensim/corpora/dictionary.py (hunk 1 above)
Comment (menshikh-iv) [reply gensim/corpora/dictionary.py]: Fixed

File: gensim/corpora/wikicorpus.py (hunk 2 above)
Comment (menshikh-iv) [reply gensim/corpora/wikicorpus.py]: fixed

File: gensim/corpora/wikicorpus.py (hunk 3 above)
Comment (menshikh-iv) [reply gensim/corpora/wikicorpus.py]: fixed
